js/donations-visualiser.js

```diff
@@ -190,6 +190,9 @@ var data_request = d3.json("data/all_data.json")
         $("#loading-modal").modal('hide');
         processData(data);
 
+        // zoom out a ways initially
+        zoomTo(0.5).event(container);
```

Comment:
it does zoom twice.. in the next PR I attempt to blend them as much as possible, given that each render is unique..